q2_fragment_insertion/_format.py

```diff
@@ -19,3 +20,26 @@ def sniff(self):
 
 PlacementsDirFmt = model.SingleFileDirectoryFormat(
     'PlacementsDirFmt', 'placements.json', PlacementsFormat)
+
+
+class RAxMLinfoFormat(model.TextFileFormat):
+    def sniff(self):
+        pplacer_pattern = '.* RAxML version ([^ ]+)'
+        with open(str(self), 'r') as f:
+            for line in f.readlines():
+                # in Greengenes and Silva cases, file starts with a lot of
+                # those warnings which we can use to "sniff" the file format
+                if line.startswith('IMPORTANT WARNING: Sequences '):
```

Comment:
I don't think this line is essential in a RAxML info file. Basically, it only happens when there are columns (sites) with identical characters, or rows (taxa) with identical sequences. I think keeping the "RAxML version" check is sufficient.